docpages/example_programs/music_and_audio/oggopus.md

```diff
@@ -2,11 +2,10 @@
 
 This example shows how to stream an Ogg Opus file to a voice channel. This example requires some additional dependencies, namely `libogg` and `opusfile`.
 
-~~~~~~~~~~~~~~~~~~~~~~~~~{.cpp}
+~~~~~~~~~~~~~~~~~~~~~~~~~cpp
 #include <dpp/dpp.h>
```

Comment:
why are these in the markdown? all these were separated out, and will build because the builder includes opus, libmp3, spdlog etc

Reply:
they were already moved? e.g.
https://github.com/brainboxdotcc/DPP/blob/master/docpages/example_code/oggopus.cpp
https://github.com/brainboxdotcc/DPP/blob/master/docpages/example_code/mp3.cpp

Reply:
There are two examples in oggopus.md the mp3.cpp is okay.